demisto_sdk/commands/validate/tests/GR_validators_test.py

```diff
@@ -529,3 +535,97 @@ def test_IsUsingUnknownContentValidator__different_dependency_type__list_files(
         [repo_for_test.packs[pack_index]]
     )
     assert len(results) == expected_len_results
+
+
+@pytest.fixture
+def repo_test_from_version(graph_repo: Repo):
+    # Repo which contains 1 pack
+
+    # Pack 1 - script uses another script (relationship)
+    pack_1 = graph_repo.create_pack("Pack2")
+    pack_1.create_script(name="FirstScript")
+    pack_1.create_script(
+        name="SecondScript",
+        code='demisto.execute_command("FirstScript", dArgs)',
```

Comment:
I'd recommend going with `Script1` and `Script2` similar to `Pack1` above, but won't block the PR on this.